start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+function wait_for () {
+  local host=$1
+  local port=$2
+
+  while ! nc -z -w 5 "$host" "$port"; do
+    sleep 1
+  done
+}
+
+wait_for solr 8983
+wait_for db 5432
+
+# Even though solr is listening, it needs a moment before the core status
+# check will return successfully.
+sleep 1
+
+# Check if the solr core exists.
+if ! (curl --get --fail --silent http://solr:8983/solr/admin/cores \
+  --data-urlencode action=status \
+  --data-urlencode core=inventory | grep -q segmentsFileSizeInBytes); then
+
+  # Create the solr core
+  curl -v --get --fail --silent http://solr:8983/solr/admin/cores \
+    --data-urlencode action=create \
+    --data-urlencode name=inventory \
+    --data-urlencode configSet=ckan2_5
+
+  # Reload the core
+  curl -v --get --fail --silent http://solr:8983/solr/admin/cores \
+    --data-urlencode action=reload \
+    --data-urlencode core=inventory
+fi
+
+# Run migrations
+paster --plugin=ckan db upgrade -c /etc/ckan/production.ini
```

Comment:
The paster upgrade is failing with 
```
app_1        | ImportError: No module named pytz
```

Reply:
I found `pytz` referenced only in `requirements-freeze.txt`. I tried `make update-dependencies; make requirements; make build; make up` and although I can no longer find a reference to `pytz` in the requirements file, the paster upgrade fails with the same error. Any ideas?